src/sidebar/components/test/ToastMessages-test.js

```diff
@@ -90,6 +90,17 @@ describe('ToastMessages', () => {
       assert.calledOnce(fakeToastMessenger.dismiss);
     });
 
+    it('should set a screen-reader-only class on `visuallyHidden` messages', () => {
```

Comment:
I typically avoid tests on element class names like the plague, but I'm not convinced there's a better way here.